src/shared/previewUtils.ts

```diff
@@ -162,26 +162,33 @@ export class PreviewUtils {
    * Generates the proper set of arguments to be used for launching desktop browser and navigating to the right location.
    *
    * @param ldpServerUrl The URL for the local dev server
+   * @param username User that is initiating the local preview
    * @param appId An optional app id for a targeted LEX app
    * @param targetOrg An optional org id
    * @param auraMode An optional Aura Mode (defaults to DEVPREVIEW)
    * @returns Array of arguments to be used by Org:Open command for launching desktop browser
    */
-  public static generateDesktopPreviewLaunchArguments(
+  public static async generateDesktopPreviewLaunchArguments(
     ldpServerUrl: string,
+    username: string,
```

Comment:
This is sort of a "one of these things is not like the other", in that the rest of the input parameters are more or less directly plugged into the functions of this method, whereas `username` is indirectly used as a lookup.

Should we retrieve the Entity ID beforehand, and pass that value in as the input parameter?

Reply:
Seems like it would be more consistent to look up Entity ID (in some capacity) in `app.ts`, somewhere around where the `ldpServerUrl` is [determined](https://github.com/salesforcecli/plugin-lightning-dev/pull/89/files#diff-4d3aa00584581c9698a9cb2dedc862456f2b15805536ccb7460a8869c80e8839R206).